### ISSUE TITLE
Kafka authorization: allow any host

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-broker#254]
**Describe the issue**
The current way to authorize producers and consumers to publish or read needs the host ip address to be specified. Example, if Bob is the producer and Alice the consumer:

```shell
fink_kafka --authorization -a -u Bob -H <ip_host> -p -t test_topic
fink_kafka --authorization -a -u Alice -H <ip_host> -c test-group -t test_topic
```

While for producer the `ip_host` is the one of the Kafka cluster producing the streams (hence easy to know), the `ip_host` of Alice is unclear (consumers can come from anywhere and change their location over time). Hence, we should have a way of specifying any hosts (currently, `-H '*'` does not work). The security would not be compromised as the consumer would still need to authenticate with username and password, and kafka stores consumer provenance.

Action item: Allow any host ip to be specified when authorizing producers and consumers.

**Comment by cAbhi15** [comment]
Thanks @JulienPeloton for finding this issue.
I'll try to fix it soon.

**Comment by JulienPeloton** [comment]
Fixed in #255.
